src/custom/custom.test.ts

```diff
@@ -0,0 +1,59 @@
+import { describe, expect, test } from "vitest";
+import { compressToCustom, decompressFromCustom } from ".";
+import { test_longString_fn } from "../__tests__/testFunctions";
+import {
+    test_hw,
+    test_repeat,
+    test_tattooSource,
+    test_complicated,
+} from "../../tests/testValues";
+
+describe("custom", () => {
```

Comment:
Can you follow the pattern of other tests - make an arrow function for each of the source dictionaries, the long text can be added to the main testFunctions so all the compression functions can be tested against it :-)

Reply:
Sorry, I thought I followed the pattern I saw in `encodedURI.test.ts`.
I am not sure what I should change here.

Reply:
Ah, I get what you mean - the proper way to do it would be something like this -

```ts
describe("custom", () => {
	describe("hex", () => {
        const hex    = "0123456789ABCDEF";
        const compress = (input) => compressToCustom(input, hex);
        const decompress = (input) => decompressFromCustom(input, hex);

        runTestSet<string>(compress, decompress, "");
    });

	describe("base62", () => {
        const base62 = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
        const compress = (input) => compressToCustom(input, base62);
        const decompress = (input) => decompressFromCustom(input, base62);

        runTestSet<string>(compress, decompress, "");
    });
});
```

The nested `describe` groups by type, and the arrow function lets you pass that second argument transparently - hope that makes more sense :-)